deploy/manifests/dev-v2/us-east-2/cluster/external-dns/deploy-patch.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-dns
+  namespace: external-dns
+spec:
+  selector:
+    matchLabels:
+      app: external-dns
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      containers:
+        - name: external-dns
+          args:
+            - --source=service
+            - --source=ingress
+            - --domain-filter=dev.cid.contact
+            - --provider=aws
+            - --policy=upsert-only
+            - --aws-zone-type=public
+            - --registry=txt
+            - --txt-owner-id=Z0346653G546POP574CS # dev.cid.contact hosted zone ID; see terraform output `dev_cid_contact_zone_id`
```

Comment:
This will interfere with the [existing controller](https://github.com/filecoin-project/storetheindex/blob/main/deploy/manifests/dev/us-east-2/cluster/external-dns/deploy-patch.yaml#L27) that uses the same hosted zone in `dev`. If we are to use external DNS in dev-v2 then my suggestion is to delegate name servers for a new subdomain and use a dedicated hosted zone ID instead.

If these manifests are already applied, can I ask you to makes sure this is updated please? Otherwise I am slightly worried about route 53 change race condition by the two controllers.

Reply:
@masih external dns has been disabled until I can complete this ticket.
https://github.com/filecoin-project/gitops-root/issues/196
 
dev-2 will get it's own zone. if the current plan is followed.

I am going to remove external dns to avoid confusion.